src/components/account/OrderScreen.js

```diff
@@ -1,10 +1,6 @@
 import React, { useEffect, useContext } from 'react';
-import {
-  View,
-  Image,
-  StyleSheet,
-  FlatList,
-} from 'react-native';
+import { View, StyleSheet, FlatList } from 'react-native';
+import { FastImage } from 'react-native-fast-image';
```

Comment:
Please refer to the official documentation of [react-native-fast-image](https://github.com/DylanVann/react-native-fast-image#readme) 

Correct way of importing `FastImage` component is 
```
import FastImage from 'react-native-fast-image'
```